### PR TITLE
Feature/multi node refactor

### PR DESCRIPTION
**Pull request by lucbel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This will compile the project and download the web driver for your operating sys
 
 The chromedriver will be downloaded in your target under target/driver/. Now you can either run ```mvn test -P integration-test``` or run the 
 ```
-com.naturalautomation.JBehaveStoriesIT
+com.com.naturalautomation.JBehaveStoriesIT
```
Comment (cbmarc) [inline README.md:29]: "com.com" fix pls

File: natural-automation-test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>com.fullstackmarc</groupId>
+            <artifactId>natural-automation</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
```
Comment (cbmarc) [inline natural-automation-test/pom.xml:23]: Plugins should not be here, at list not configured here.

File: natural-automation/src/main/java/com/naturalautomation/Config.java [hunk 3]
```diff
@@ -10,7 +10,7 @@
 import java.io.IOException;
 
 @SpringBootApplication
-@ComponentScan(basePackages = "com.naturalautomation")
+@ComponentScan(basePackages = "com.com.naturalautomation")
```
Comment (cbmarc) [inline natural-automation/src/main/java/com/naturalautomation/Config.java:13]: "com.com"

**Review by cbmarc — CHANGES_REQUESTED** [review]
Please review

**Comment by lucbel** [comment]
I rebased the main branch

**Pull request closed without merging**